<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-CSMSJ8J" name="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32" tests="10" failures="1" timestamp="2020-12-28T17:16:37 PKT" time="38.555" errors="0">
  <testcase name="Test_ForgotPhoneNumber_Button_is_clickable_On_LoginPage_23" time="2.277" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Logo_24" time="5.762" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Forgot_Phone_Number_Text_25" time="2.155" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Enter_Register_Email_Text_26" time="2.197" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_To_ClickOn_SendButton_without_Entering_Email_27" time="1.196" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Email_Validation_28" time="3.995" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Email_Not_Link_Validation_ErrorMessage_29" time="5.424" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_ValidEmail_are_accepted_and_SendMessage_30" time="9.790" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32">
    <failure type="java.lang.AssertionError" message="expected [SMS send successfully at 009xxxxxx78] but found [SMS send successfully at 009xxxxxx72]">
      <![CDATA[java.lang.AssertionError: expected [SMS send successfully at 009xxxxxx78] but found [SMS send successfully at 009xxxxxx72]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32.Test_ForgotPhoneNumberPage_ValidEmail_are_accepted_and_SendMessage_30(ForgotPhoneNumberPage_TestCases_23_32.java:178)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Test_ForgotPhoneNumberPage_ValidEmail_are_accepted_and_SendMessage_30 -->
  <testcase name="Test_ForgotPhoneNumberPage_verify_ResendMessageButton_is_working_31" time="2.275" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
  <testcase name="Test_ForgotPhoneNumberPage_Message_SendCross_Button_BringBack_User_toLoginPage_32" time="3.484" classname="com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32"/>
</testsuite> <!-- com.InfoCabs.testcases.ForgotPhoneNumberPage_TestCases_23_32 -->
